public_suffix_list.dat

```diff
@@ -13948,6 +13948,10 @@ sopot.pl
 // Submitted by Cedric Dubois <cedric.dubois@team.blue>
 site.tb-hosting.com
 
+// TechFX : https://techfx.com.br
+// Submitted by Alan Sikora <alan@techfx.com.br>
+techfx.com.br
```

Comment:
You seem to be using this domain. Warning: If it becomes a public suffix you cannot set cookies on it anymore.